source/_integrations/homekit.markdown

```diff
@@ -564,6 +564,8 @@ Pairing works fine when the filter is set to only include `demo.demo`, but fails
 
 Check if the domain of your entity is [supported](#supported-components). If it is, check your [filter](#configure-filter) settings. Make sure the spelling is correct, especially if you use `include_entities`.
 
+If your sensor is not visible, make sure the `unit_of_measurement` is set correctly as per [Valid units of measurement](https://github.com/home-assistant/core/blob/d7ac4bd65379e11461c7ce0893d3533d8d8b8cbf/homeassistant/const.py#L384). For example the correct unit of measurement for a temperature sensor is either "°C" or "°F", and for a humidity sensor it is "%". Use the [Customizing entitities](https://www.home-assistant.io/docs/configuration/customizing-devices/) to change the `unit_of_measurement` for an entity.
```

Comment:
We can't link to the code directly here since it will change over time which will make the link invalid

Reply:
We should not link to code either way. And end user has no clue what they will be looking at.

Additionally, don't use full URLs and don't recommend/advise the customize method.